File: challenges/2-idade-em-dias/go/jessicamosouza/idadeEmDias.go
package main

import "fmt"

func main() {
	var idade int

	fmt.Print("Idade em dias: ")
	fmt.Scan(&idade)

	anos := idade / 365
	meses := (idade % 365) / 30
	dias := (idade % 365) % 30

	fmt.Printf("%v ano(s) \n%v mes(es) \n%v dia(s)", int(anos), meses, dias)
}

File: challenges/2-idade-em-dias/java/nformenton/IdadeEmDias.java
import java.util.Scanner;

public class IdadeEmDias {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        
        System.out.print("Digite sua idade em dias: ");
        int dias = scan.nextInt();

        int ano = 365;
        int meses = 30;

        int qtdAno = dias/ano;
        int qtdMeses = (dias % ano)/meses;
        int qtdDias = (dias % ano) % meses;

        System.out.println(qtdAno + " anos(s)");
        System.out.println(qtdMeses + " mes(es)");
        System.out.println(qtdDias + " dia(s)");
    }
}
